modules/gittar/pkg/gitmodule/repo_edit.go

```diff
@@ -35,6 +36,13 @@ const (
 	EDIT_PATH_TYPE_BLOB EditPathType = "blob"
 )
 
+var (
+	NotSupportType      = errors.New("not support path type")
+	NotSupportAction    = errors.New("not support action")
+	FileAlreadyExists   = errors.New("存在同名文件")
+	FolderAlreadyExists = errors.New("存在同名文件夹")
+)
+
```

Comment:
suggest don't specifically declare global variables here

Reply:
refer to https://github.com/uber-go/guide/blob/master/style.md#error-types
![image](https://user-images.githubusercontent.com/23724009/118125264-5523cb00-b429-11eb-9a88-f7dd86fa36f3.png)